install/helm/agones/templates/service/allocation.yaml

```diff
@@ -160,6 +160,9 @@ spec:
 {{- end }}
 {{- if .Values.agones.allocator.annotations }}
 {{- toYaml .Values.agones.allocator.annotations | nindent 8 }}
+{{- end }}
+{{- if .Values.agones.allocator.labels }}
```

Comment:
Looks good - just checking if you had a chance to do any manual testing on this? (I don't think there's any better way to actually test this).

Easiest way might be to use `make install` and pass in some extra `HELM_ARGS` and make sure the Deployment labels are good.

Reply:
I have validated the labels manually per your guidance

Reply:
Thanks for the validation - but I just realised after looking at it, that these values are showing up in the `annotations:` section.

We should move this code up to above line 152, so it sits under `labels:` - then it should be good to go.

Reply:
The values have been moved to the labels section